_observing-your-data/ad/index.md

```diff
@@ -160,13 +177,54 @@ If the actual total number of unique entities is higher than the number that you
 This formula serves as a starting point. Make sure to test it with a representative workload. You can find more information in the [Improving Anomaly Detection: One million entities in one minute](https://opensearch.org/blog/one-million-enitities-in-one-minute/) blog post.
 {: .note }
 
-#### (Advanced settings) Set a shingle size
+### (Advanced settings) Set a shingle size
 
 Set the number of aggregation intervals from your data stream to consider in a detection window. It’s best to choose this value based on your actual data to see which one leads to the best results for your use case.
 
-The anomaly detector expects the shingle size to be in the range of 1 and 60. The default shingle size is 8. We recommend that you don't choose 1 unless you have two or more features. Smaller values might increase [recall](https://en.wikipedia.org/wiki/Precision_and_recall) but also false positives. Larger values might be useful for ignoring noise in a signal.
+The anomaly detector requires the shingle size to be between 1 and 128. The default is `8`. Use `1` only if you have at least two features. Values less than `8` may increase [recall](https://en.wikipedia.org/wiki/Precision_and_recall), but also increase false positives. Values greater than `8` may be useful for ignoring noise in a signal.
+
+To set the shingle size, select **Show** on the **Advanced settings** pane. Enter the desired size in the **intervals** field.
+
+### (Advanced settings) Set an imputation option
```

Comment:
@kaituo Where does the user go in OpenSearch Dashboards to set this option?

Reply:
They can do it under Advanced settings section, same as shingle size above.